### PR TITLE
[RFR] Add characteristics

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -75,7 +75,7 @@ test-frontend-unit: ## Run the frontend application unit tests
 		"./src/app/js/**/*.spec.js"
 
 test-frontend-functional: ## Run the frontend application functional tests
-	NODE_ENV=test ${MAKE} build-frontend
+	# NODE_ENV=test ${MAKE} build-frontend
```
Comment (djhi) [inline makefile]: woops

File: src/app/js/i18n/translations/en.js [hunk 2]
```diff
@@ -148,4 +148,8 @@ export default {
     after_field: 'After %{field}',
     first_position: 'At first position',
     position: 'Position',
+    add_characteristic: 'add characteristic',
+    label: 'label',
+    value: 'value',
+    'add-field-to-resource': 'add field',
```
Comment (djhi) [inline src/app/js/i18n/translations/en.js]: change dashes for underscores and loose the quotes

File: src/app/js/lib/DialogButton.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+import FlatButton from 'material-ui/FlatButton';
+import Dialog from 'material-ui/Dialog';
+import IconButton from 'material-ui/IconButton';
+import classnames from 'classnames';
+import CircularProgress from 'material-ui/CircularProgress';
+
+import ButtonWithStatus from './ButtonWithStatus';
+import { polyglot as polyglotPropTypes } from '../propTypes';
+
+export const DialogButtonComponent = ({
```
Comment (djhi) [inline src/app/js/lib/DialogButton.js]: I would rename this component to `ButtonWithDialog`. I first thought that it was a button for inside the dialog

File: src/app/js/public/dataset/Dataset.js [hunk 4]
```diff
@@ -90,6 +92,9 @@ export class DatasetComponent extends Component {
                         showing: polyglot.t('showing'),
                     }}
                 />
+                <CardActions style={styles.actions}>
```
Comment (djhi) [inline src/app/js/public/dataset/Dataset.js:95]: Isn't this only available to Julia ?

**Review by djhi — CHANGES_REQUESTED** [review]

File: src/app/js/public/dataset/Dataset.js (hunk 4 above)
Comment (ThieryMichel) [reply src/app/js/public/dataset/Dataset.js:95]: It's the AddCharacteristic responsibility, to show itself or not.

**Review by djhi — APPROVED** [review]

**Pull request merged**